### PR TITLE
Remove unused files in Website doc

### PR DESCRIPTION
**Pull request by ChaiBapchya**
## Description ##
After the revamping of the MXNet website, we no longer need lots of docs from 
```
python/mxnet/ndarray_doc.py
python/mxnet/symbol_doc.py
```

Only the _build_doc functions described in these 2 docs are used. Hence keeping that and yanking the rest.

Realized this while closing my previous PR - #14243 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- [x] Code is well-documented: 
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- modified:    python/mxnet/ndarray_doc.py
- modified:    python/mxnet/symbol_doc.py

@aaronmarkham @sojiadeshina Please confirm

### CODE REVIEW

**Comment by ChaiBapchya** [comment]
Turns out the CI is failing coz register.py uses _build_doc
Eg
https://github.com/ChaiBapchya/incubator-mxnet/blob/f1b7b491bb5b1c7c9f7c3fa893af0abdc151cb0e/python/mxnet/symbol/register.py#L29

Does this mean I can remove rest of the contents of the symbol_doc and ndarray_doc files but keep the _build_doc function since it is being used here 
OR
Should I remove the function call to _build_doc (so that the CI passes) - I just want to make sure doc building doesn't depend on the 2 files in any way.

@aaronmarkham @sojiadeshina can either of you confirm? Thanks

**Comment by ChaiBapchya** [comment]
I've gone ahead and made the change.
Kept _build_doc and yanked the rest of the functions (doc related) which aren't used anymore.
If the CI passes, would ping folks for review & merge.

**Comment by ChaiBapchya** [comment]
@mxnet-bot run ci [all]

**Comment by leezu** [comment]
@mxnet-bot run ci [all]

File: ci/safe_docker_run.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def _trim_container_id(cid):
         return cid[:12]
 
     def __init__(self):
-        self._docker_client = docker.from_env()
+        self._docker_client = docker.from_env(timeout=None)
```
Comment (leezu) [inline ci/safe_docker_run.py:57]: @marcoabreu FYI after rebuilding our Unix CPU Jenkins Slave AMI via the scripts in https://github.com/apache/incubator-mxnet-ci/ we intermittently run into https://github.com/docker/docker-py/issues/2266 .

The rootcause is that when a new instance is spawned, booted, Docker agent started and Jenkins immediately requests starting a container, docker may take longer than 60 seconds to finish the startup. In the [log](http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fedge/detail/PR-16722/8/pipeline), line 294 to 295 you can see that the startup took 70 seconds, triggering the timeout.

The issue is tracked upstream at https://github.com/docker/docker-py/issues/2266 and appears to be a regression in Docker. (Note that we're using a 2 year old Docker version previously, as the AMI hasn't been regenerated.) 

Considering we're talking about a socket connecting to localhost, we have no risk of connection issues and do not need to use a timeout.

Added the fix to this PR, as this PR experienced the issue and is ready to be merged.

**Pull request merged**